packages/editor/src/components/visual-editor/edit-template-blocks-notification.js

```diff
@@ -55,9 +55,13 @@ export default function EditTemplateBlocksNotification( { contentRef } ) {
 				return;
 			}
 
-			if ( ! event.target.classList.contains( 'is-root-container' ) ) {
+			if (
+				! event.target.classList.contains( 'is-root-container' ) ||
+				event.target.dataset?.type === 'core/template-part'
+			) {
 				return;
 			}
+			event.stopImmediatePropagation();
```

Comment:
Not a blocker, just curious: why do we need to stop immediate propagation? I'd assumed that if it was the template part block, we wouldn't wind up here as there's the early return 🤔

In practice, I didn't run into any issues with this change.

Reply:
Good question. It's an artefact from a previous version which didn't have the `event.target.dataset?.type === 'core/template-part'` check. 

I'll remove to avoid confusion. 👍🏻 

Thank for testing!